src/components/SpeakerCard/SpeakerCard.scss

```
@@ -0,0 +1,78 @@
+@use "~stylesheets/typography";
+
+.SpeakerCard {
+  padding: 4% 5%;
+  margin: auto;
```

```suggestion
  margin: 4% 5%;
```
Personally, padding usually applies to interior space, while margins apply to spacing between components. In this iteration, it looks like the padding is actually empty, so changing it to margins might make more sense. Note that if this is implemented, by comments below about `align-content` and align`items` may not apply.

---

So I went with this approach since this allowed us to center the boxes within the flex container while still maintaining some distance between each card component. Do you have any other suggestions?